scripts/release.sh

```diff
@@ -29,7 +29,8 @@ else
 fi
 
 if [[ $( git branch --show-current) != "main" ]]; then 
-    TAG=$TAG"-beta"
+    COMMIT=$(git rev-parse --short HEAD)
+    TAG=$TAG"-$COMMIT"
```

Comment:
changes the name of a beta tag to the commit id to get rid of collisions when working of simultaneous branches